protocols/v2/subprotocols/common-messages/src/setup_connection.rs

```diff
@@ -71,11 +71,11 @@ impl<'decoder> SetupConnection<'decoder> {
             Protocol::MiningProtocol => {
                 let available = available_flags.reverse_bits();
```

Comment:
Could you please add an explanation above the L72 describing what are the checks we are doing?

Reply:
Aren't these self-explanatory, as in taking the first bit and checking the condition based on whether it's set or not? However, I will still add documentation if it's unclear to others as well.

Reply:
Yea, I think it would be better to document those. Not necessarily line by line, be the general approach